torch/onnx/_internal/fx/fx_onnx_interpreter.py

```diff
@@ -520,6 +541,9 @@ def run(
                     fx_name_to_onnxscript_value,
                 )
 
+        with diagnostic.log_section(logging.DEBUG, "ONNX Graph:"):
+            diagnostic.debug("```\n%s\n```", onnxscript_graph.torch_graph)
```

Comment:
What's the difference to make this one not LazyString?

Reply:
logging api, adopted from python `logging`, lazy formats the string. Hence in this case as long as getattr `onnxscript_graph.torch_graph` is cheap it is fine. The real formatting happens later.

On the contrary. If let's say your argument is `graph_module.print_readable(print=False)`, which runs codegen when called, you'd need to put it behind `LazyString`.